src/bundle.c

```diff
@@ -552,13 +560,29 @@ int install_bundles_frontend(char **bundles)
 
 	for (; *bundles; ++bundles) {
 		bundles_list = list_prepend_data(bundles_list, *bundles);
+		if (*bundles) {
+			if (bundles_list_str) {
+				string_or_die(&bundles_list_str, "%s, %s", bundles_list_str, *bundles);
```

Comment:
You will want to `free(bundles_list_str)` before this line to avoid a memory leak.